WebExample/tests/styles.spec.ts

```diff
@@ -0,0 +1,66 @@
+import {test, expect} from '@playwright/test';
+import type {Page} from '@playwright/test';
+import * as TEST_CONST from '../../testConstants';
+import {setupInput} from './utils';
+
+const testMarkdownContentStyle = async ({styleName, style, page}: {styleName: string; style: string; page: Page}) => {
+  const inputLocator = await setupInput(page);
+
+  const elementHandle = inputLocator.locator('span', {hasText: styleName}).last();
+
+  let elementStyle;
+
+  if (elementHandle) {
+    await elementHandle.waitFor({state: 'attached'});
+
+    elementStyle = await elementHandle.getAttribute('style');
+  }
+  expect(elementStyle).toEqual(style);
+};
+
+test.beforeEach(async ({page}) => {
+  await page.goto(TEST_CONST.LOCAL_URL, {waitUntil: 'load'});
+  await page.click('[data-testid="reset"]');
+});
+
+test.describe('markdown content styling', () => {
+  test('bold', async ({page}) => {
+    await testMarkdownContentStyle({styleName: 'bold', style: TEST_CONST.MARKDOWN_STYLE_DEFINITIONS.bold.style, page});
```

Comment:
I'm not a huge fan of `TEST_CONST.MARKDOWN_STYLE_DEFINITIONS`. There's nothing wrong with tests containing string literals in multiple copies.

This test should contain `"Hello *world*!"` string as well as `fontWeight: 'bold'` somewhere inside.

Reply:
in terms of `"Hello *world*!"` - sure, but I think having styles defined in a constant is the right way to handle this, If you try checking style definition in multiple cases and/or you are using this styling in a library itself, this might get desynchronized, if someone decides to change it at some point